script/convert-grammars

```diff
@@ -148,6 +148,17 @@ def load_grammar(path)
   case File.extname(path.downcase)
   when '.plist', '.tmlanguage'
     Plist::parse_xml(path)
+  when '.yaml-tmlanguage', '.sublime-syntax'
+    content = File.read(path)
+    # Attempt to parse YAML file even if it has a YAML 1.2 header
+    if content.lines[0] =~ /^%YAML[ :]1\.2/
+      content = content.lines[1..-1].join
+    end
+    begin
+      YAML.load(content)
+    rescue Psych::SyntaxError => e
+      $stderr.puts "Failed to parse YAML grammar '#{path}'"
```

Comment:
Most syntax files declare a YAML 1.2 syntax (`%YAML 1.2`) although they are YAML 1.1 compatible.
Here, the YAML version header is stripped off before parsing. If parsing fails, it displays a warning.